### ISSUE TITLE
Error in plot_process_tree js callback to copy selection to python variable

### ISSUE THREAD

**Issue by marcohenriques** [microsoft/msticpy#186]
The JS callback for the _plot_process_tree_ doesn't have the `var` for the variables **out_str** and **py_str**, causing the execution to fail, telling the variables were not found.

**Comment by ianhelle** [comment]
Thanks @marcohenriques - good timing since I'm currently working on some updates to process tree to get it working with MDE and more generic event sets.
One thing to note about this functionality is that it only works in Jupyter classic - even when it has correct JS syntax!  I'm trying to come up with a generic mechanism to have Bokeh JS objects communication values back to to Ipykernel using ipwidgets.

PS not sure how the custom JS code got into this state - I certainly tested it working when I first built it. Hmm!

**Comment by marcohenriques** [comment]
Yeah, I actually saw the PR, and I wasn't sure if I should create a new issue or just add a comment on that one. And btw, make it work with MDE would be great, I'm currently using the mdatp API for the queries (I tried to use msticpy, but didn't find a way to connect using certificates and key.pem, even tried to "override" the connect method, but with no sucess). Based on the outputs from it I'm trying to build a process tree, I managed to map some fields, but there were others that I had to hard-code because I think they were not applicable. Which isn't a very clean way to do it.

Also, I faced some other issues in this process tree plot so started to do some modifications (based on package code), one of them was on the `_create_fill_map`, because I had some python `None`'s on the label column, and it failed because the np.nan/np.isnan doesn't detect this, I used the pandas `pd.isna` instead.

Other thing I did was to give a the ability to customize which columns to show when we hover and the ones we show on the tables.

This is just my feedback from using it, hope it can help somehow. Thanks
